app/test.jsx

```diff
@@ -131,8 +137,9 @@ class App extends React.Component {
                         &nbsp;
                         <b>low:</b> <input id="low" type="number" value={low === undefined ? '' : low} onChange={this.onHighLowChange} />,
                         <b>high:</b> <input id="high" type="number" value={high === undefined ? '' : high} onChange={this.onHighLowChange} />;
-                        No PriceLine: <input type="checkbox" onChange={this.onPriceLineDisableChange} />
-                        Relative: <input type="checkbox" onChange={this.onRelativeChange} />
+                        No PriceLine: <input type="checkbox" checked={hidePriceLines === undefined ? '' : hidePriceLines} onChange={this.onPriceLineDisableChange} />
```

Comment:
Is it better to check your variable with typeof? 
`typeof hidePriceLines != 'undefined'`

This will fail of you have not declared the variable 
`hidePriceLines` with error `Uncaught ReferenceError: params is not defined`

Reply:
@jbaek7023 I'm not sure I get your point. It's `typeof t === "undefined"` vs simply `t === undefined`. Why will I get a `ReferenceError`?

Reply:
@brucebinary  It's for the safer usage and nothing else. 👍 

Reference: https://stackoverflow.com/questions/16453569/comparing-undefined-value-in-javascript

Reply:
if that is the reason I prefer to have that error thrown. I don't see how is it safer. If I'm referencing something that doesn't exist anywhere in context I do not expect it to work. Here, as I am accessing the values from an object, it will always be either `undefined` or something else.